OTVision/track/track.py

```diff
@@ -122,24 +122,32 @@ def main(
         metadata = preprocessed.metadata
         detections = frame_group.to_dict()
         _check_and_update_metadata_inplace(otdict=detections)
+        tracker_data: dict = {
+            dataformat.NAME: "IOU",
+            dataformat.SIGMA_L: sigma_l,
+            dataformat.SIGMA_H: sigma_h,
+            dataformat.SIGMA_IOU: sigma_iou,
+            dataformat.T_MIN: t_min,
+            dataformat.T_MISS_MAX: t_miss_max,
```

Comment:
Is the tracker data entry `utm: bool` still relevant?

Reply:
No, it is currently not relevant.